_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "emldown: From machine readable EML metadata to a pretty documentation website"
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+  - name: Andrew MacDonald
+    url: http://aammd.info/
+  - name: Kara Woo
+    url: http://karawoo.com/
+  - name: Carl Boettiger
+    url: http://www.carlboettiger.info/
+  - name: Jeff Hollister
+    url: https://jwhollister.com/
+date: "2017-08-01"
+output:
+  md_document:
+    variant: markdown_github
+    toc: true
+---
+
+We all know we should document our data, and some documentation is better than no documentation.  Even better, it's best practice to write metadata, i.e. structured documentation, for a dataset based on a standard machine-readable metadata format rather than in an unstructured document. A well-established standard is the [Ecological Metadata Language (EML)](https://knb.ecoinformatics.org/#external//emlparser/docs/index.html). If you follow the standard properly, you're sure to have documented your data well, which gives you a good feeling. Moreover, this will allow easier sharing of that dataset later. 
```

Comment:
"Even better, it's best practice" 

better and best in the same sentence reads a bit awkward.